app/cards/page.tsx

```diff
@@ -0,0 +1,71 @@
+"use client";
+import Link from "next/link";
+import { useState, useEffect } from "react";
+
+// export async function getStaticProps() {
+//   const allCardsData = localStorage.getItem("cards");
+//   console.log(allCardsData);
+//   return {
+//     props: {
+//       allCardsData,
+//     },
+//   };
+// }
+
+export default function Cards() {
+  const [cards, setCards] = useState([]);
+
+  useEffect(() => {
+    let allCardsData = JSON.parse(localStorage.getItem("cards"));
+    setCards(allCardsData);
+
+    console.log(allCardsData);
+  }, []);
+
+  let Section;
+
+  if (!cards) {
+    Section = <section>Ви не створили жодної картки</section>;
+  } else {
+    Section = (
+      <section>
+        <h2 className="mt-10 hover:bg-sky-700 text-grey font-bold">
+          Ваша дитина
+        </h2>
+        <ul>
+          {cards.map(
+            (card: {
+              id: string;
+              name: string;
+              age: number;
+              gender: string;
+            }) => (
```

Comment:
Після винесеного типу можна буде:

```suggestion
            (card: Card) => (
```